Kalpa/light-inline.svg

```diff
@@ -0,0 +1,13 @@
+ <metadata>
+  <rdf:RDF>
+   <cc:Work rdf:about="">
+    <dc:format>image/svg+xml</dc:format>
+    <dc:type rdf:resource="http://purl.org/dc/dcmitype/StillImage"/>
+   </cc:Work>
+  </rdf:RDF>
+ </metadata>
+ <path d="m8.7034 0.0038424c-0.098412 0.013338-0.19369 0.061445-0.26573 0.13891l-3.3344 3.5711c-0.14019 0.15062-0.15983 0.37835-0.048569 0.55273l4.203 6.5865c-0.67883 1.2333-1.2159 2.5589-1.5858 3.9502l-7.3289 1.7624c-0.20397 0.04885-0.34878 0.23532-0.34287 0.44855l0.13715 4.9138c0.005714 0.20782 0.15241 0.38552 0.35145 0.4283l7.206 1.5685c0.4259 1.5871 1.057 3.0837 1.8772 4.4537l-3.9659 6.5171c-0.11063 0.18213-0.079203 0.41504 0.074286 0.56141l3.5259 3.3772c0.14828 0.14159 0.37566 0.16415 0.5486 0.0521l6.3803-4.1788c1.2958 0.70906 2.6871 1.2583 4.1516 1.6148l1.7201 7.3331c0.04841 0.20787 0.2337 0.35032 0.44288 0.34438l4.8516-0.13891c0.20397-0.0058 0.37694-0.152 0.42002-0.35595l1.5801-7.4402c1.4869-0.45207 2.8868-1.1024 4.1716-1.9187l6.5146 4.0659c0.17915 0.11235 0.41204 0.08303 0.55716-0.07235l3.3344-3.5711c0.14106-0.15106 0.1601-0.37882 0.04857-0.55274l-2.443-3.8315-0.74861 0.26335c-0.10916 0.03839-0.22968-2.38e-4 -0.29715-0.0955 0 0-1.5517-2.1658-3.5544-4.963-2.1737 4.624-6.7659 7.8771-12.169 8.0306-3.4888 0.09906-6.7115-1.1138-9.2118-3.1949l0.70003-0.70901c5.0718 4.2027 12.594 3.9131 17.324-0.87685 4.7294-4.79 5.0181-12.409 0.86861-17.546l4.6174-4.6765a1.2931 1.3096 6.9159e-4 1 0-1.8287-1.8521l-4.6173 4.6765c-5.0718-4.2027-12.594-3.9131-17.324 0.87685-4.7294 4.79-5.0152 12.409-0.86575 17.546l-0.71146 0.71768c-1.9429-2.3468-3.1488-5.345-3.2401-8.6441-0.15833-5.7199 3.0743-10.737 7.8489-13.078l-0.10286-3.6318c-0.87145 0.33682-1.7101 0.73797-2.5058 1.2068l-0.0029-0.00296-6.6689-4.1643c-0.089794-0.055846-0.19306-0.07675-0.29144-0.063669zm15.621 8.8959c2.6233-0.013184 5.2522 0.98453 7.2803 2.9894a1.293 1.3096 6.3397e-4 0 0 0.17429 0.17653c3.9626 4.1081 3.9436 10.649-0.07714 14.721-4.0196 4.0711-10.476 4.0908-14.532 0.08104a1.293 1.3096 6.3397e-4 0 0-0.17429-0.17653c-3.9626-4.1081-3.9436-10.649 0.07714-14.721 2.0098-2.0355 4.6285-3.0573 7.2518-3.0704zm0.06286 5.2495c-1.3262 0-2.6529 0.50781-3.6573 1.5251-2.0088 2.0345-2.0088 5.3738 0 7.4084 2.0088 2.0345 5.3058 2.0345 7.3146 0 2.0088-2.0345 2.0088-5.3738 0-7.4084-1.0044-1.0173-2.3311-1.5251-3.6573-1.5251zm0 2.6016c0.65905 0 1.3183 0.2587 1.8287 0.77556 1.0206 1.0337 1.0206 2.6705 0 3.7042s-2.6367 1.0337-3.6573 0c-1.0206-1.0337-1.0206-2.6705 0-3.7042 0.51032-0.51686 1.1696-0.77556 1.8287-0.77556z" fill="#fff" stroke-width="1.4723"/>
+ <text x="46.933743" y="32.159557" fill="#ffffff" font-family="SUSE" font-size="31.174px" font-weight="600" stroke-width=".41751" style="line-height:0;paint-order:markers stroke fill" xml:space="preserve"><tspan x="46.933743" y="32.159557" fill="#ffffff" stroke-width=".41751">Kalpa</tspan></text>
```

Comment:
You should use "Path -> Join" in inkscape on this element, so that it is a path and not text